src/transformer/cli.js

```diff
@@ -55,23 +54,22 @@ export default async function ({name, transformCallback, validateCallback}) {
 	}
 
 	const spinner = ora('Transforming records').start();
-	const records = await transformCallback(fs.createReadStream(args.file));
+	const records = await transformCallback(fs.createReadStream(args.file), args.validate, args.fix);
```

Comment:
`validate` and `fix` are Melinda-specific functions. We need an extensions system for the CLI which allows defining flags to trigger post-transformation actions.